# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/no-systemd.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-256.2/src/udev/udevd.c.vanilla	2024-07-13 09:03:30.158846878 +0200
+++ systemd-256.2/src/udev/udevd.c	2024-07-13 09:03:43.731845571 +0200
@@ -341,7 +341,7 @@
         if (arg_daemonize) {
                 pid_t pid;
 
-                log_info("Starting systemd-udevd version " GIT_VERSION);
+                log_info("Starting udevd version " GIT_VERSION);
 
                 /* connect /dev/null to stdin, stdout, stderr */
                 if (log_get_max_level() < LOG_DEBUG) {
